packages/core/src/middleware/middleware-resolver.ts

```diff
@@ -28,6 +28,7 @@ class MiddlewareResolver {
     morgan: "morgan",
     helmet: "helmet",
     rateLimit: "express-rate-limit",
+    session: "exporess-session",
```

Comment:
There is a typo here: `session: "exporess-session"`

Correct is: `session: "express-session"`

Reply:
@rsaz 
Sorry. I modified.